mlkem/debug/debug.h

```diff
@@ -127,6 +162,12 @@ void mlkem_debug_print_error(const char *file, int line, const char *msg);
 
 #else /* MLKEM_DEBUG */
 
+#define CASSERT(...) \
+  do {               \
+  } while (0)
```

Comment:
Should we maybe prefix all of those with `MLKEM_DEBUG_` or `DEBUG_` to mark that they all belong to the `debug.c`?

Reply:
Yes, could do, but let's do it in a separate PR.